### PR TITLE
Image 404 Breaks Graph (Issue #1814)

### PR DESCRIPTION
**Pull request by tumri**
Fixes render error when image 404s.

### CODE REVIEW

File: src/extensions/renderer/base/images.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ BRp.getCachedImage = function( url, crossOrigin, onLoad ){
   if( cache ){
     if( !cache.image.complete ){
       cache.image.addEventListener('load', onLoad);
-      cache.image.addEventListener('error', function(){ cache.image.width = cache.image.height = 0; });
+      cache.image.addEventListener('error', function(){ cache.image.renderable = cache.image.renderable = false; });
```
Comment (maxkfranz) [inline src/extensions/renderer/base/images.js]: The duplicate assignment isn't needed with one property now

File: src/extensions/renderer/base/images.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ BRp.getCachedImage = function( url, crossOrigin, onLoad ){
 
     var image = cache.image = new Image(); // eslint-disable-line no-undef
     image.addEventListener('load', onLoad);
-    image.addEventListener('error', function(){ image.width = image.height = 0; });
+    image.addEventListener('error', function(){ image.renderable = image.renderable = false; });
```
Comment (maxkfranz) [inline src/extensions/renderer/base/images.js]: Same with this prop

File: src/extensions/renderer/canvas/drawing-nodes.js [hunk 3]
```diff
@@ -148,7 +148,9 @@ CRp.drawNode = function( context, node, shiftToOriginWithBb, drawLabel ){
   for( var i = 0; i < numImages; i++ ){
     if( ( urlDefined[i] ) && image[i].complete ){
       totalCompleted++;
-      this.drawInscribedImage( context, image[i], node, i );
+      if ( !(image[i].renderable == false) ) {
```
Comment (maxkfranz) [inline src/extensions/renderer/canvas/drawing-nodes.js]: Maybe better to name the prop `error` so it's not a double negative condition

**Comment by tumri** [comment]
Changed as requested.

**Comment by tumri** [comment]
Changed as requested.

**Comment by maxkfranz** [comment]
@tumri OK, I squashed and cherry-picked across master and unstable.  But the extra listener in the existing cache case isn't needed so I removed it.

**Pull request closed without merging**